docs/docs/guides/hosting-own-have-i-been-pwned-api.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: hosting-own-have-i-been-pwned-api
+title: Hosting own Have I Been Pwned API
+---
+
+Ory Kratos uses the Have I Been Pwned (HiBP) API, with the k-anonymity flag,
+to check if the password the user registers with has been a part of one 
+of the previous leaks.
+
+If your Kratos operates in an environment where the egress to the HiBP public
+API hosted at haveibeenpwned.com is restricted via network policies, for
+example as a result of CISO policies, you can host the HiBP API in-house
+and configure Kratos to use your own instance of the API. To do so, configure
+Kratos to use your API:
+
+```yaml title="path/to/kratos/config.yml
+selfservice:
+  methods:
+    password"
+      config"
```

Comment:
```suggestion
    password:
      config:
```

Reply:
It’s been fixed in this commit: https://github.com/ory/kratos/pull/1009/commits/fbf2bc7537d9598e704e92b2b9e35cb970107aae